src/templates/default/reviews/review_stages.html

```diff
@@ -0,0 +1,114 @@
+{% extends 'dashboard_base.html' %}
+
+{% load i18n static %}
+{% load proposals %}
+{% load compress crispy_forms_tags %}
+{% load tz %}
+{% get_current_timezone as TIME_ZONE %}
+
+{% block dashboard_tablist %}
+{% include '_includes/dashboard_tablist.html' with active='admin' %}
+{% endblock dashboard_tablist %}
+
+{% block main-content %}
+
+<h3 class="dashboard-header">
+	<span class="header-title">{% trans 'Review Stages' %}</span>
+</h3>
+
+<div class="dashboard-body">
+	<!--Hot Key Button-->
+	<div class="col-md-3">
+		{% for rs in review_stages_list %}
+		<button class="btn btn-natural-noborder hotkey" value="{{ rs }}">{{ rs }}</button>
+		{% endfor %}
+	</div>
+	<!--Review Change Setting-->
+	<div class="col-md-9">
```

Comment:
I suggest you make these two blocks (vertically) instead of two columns. Something like the following:
<img width="757" alt="圖片" src="https://user-images.githubusercontent.com/484883/97077336-823b9800-1615-11eb-918a-700c6529a16c.png">

If you want to make the change very fast, simply change `col-md-3` and `col-md-9` to `col-md-12`. But maybe you could just refine the HTML structure here, like, maybe not using `col-md-...` here and simple `div` blocks,
or change it to two
```
<div class="row">
    <div class="col-md-12">
    </div>
</div>
```
rows.

The reason why not to use column is that both blocks require longer width. The original version will not look well if my screen is not wide enough:
<img width="757" alt="圖片" src="https://user-images.githubusercontent.com/484883/97077390-e8281f80-1615-11eb-9f7b-f857603ca786.png">

Reply:
This layout is indeed more comfortable. It changed at afaf6d5af4c52c1f03a060f70e09fba06482dbaf